### ISSUE TITLE
Broken with Tox 3.2.0

### ISSUE THREAD

**Issue by wcooley** [tox-dev/tox-pip-extensions#11]
Tox 3.2.0+ with tox-pip-extensions 1.2.1 is broken due to tox-dev/tox#935 changing the default `install_command` and this having a [hardcoded](https://github.com/tox-dev/tox-pip-extensions/blob/master/tox_pip_extensions.py#L27) value.

Either the check needs to take this into account in the comparison at [line 65](https://github.com/tox-dev/tox-pip-extensions/blob/master/tox_pip_extensions.py#L65) or compare directly with `tox.config.InstallcmdOption.default`.

Also, it might be nice if it raised an exception instead of calling `sys.exit` or otherwise gave some context; it was a little mystifying at first to receive this error from seemingly out of nowhere:

```
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
tox-pip-extension(s) (venv_update) used but testenv (py3) sets install_command!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

**Comment by asottile** [comment]
The error message is saying that the `py3` environment sets `install_command` which is incompatible with the project.  I don't think the error message (or methodology) really needs to change but the vanilla argument change does seem unfortunate.

It was initially hardcoded defensively due to another plugin but that plugin no longer exists (I obsoleted it with this).

While patching this I realized why it is hardcoded, parts are replaced out of it and I wanted to be careful that features weren't silently dropped.

Making a patch for this now
